module-1/README.md

```diff
@@ -63,55 +117,255 @@ export const environment = {
+
+```typescript
+const origin = new cloudfront.CfnCloudFrontOriginAccessIdentity(this, "BucketOrigin", {
+  cloudFrontOriginAccessIdentityConfig: {
+    comment: projectName
+  }
+});
+
+const bucketPolicy = new iam.PolicyStatement()
+  .addPrincipal(new iam.CanonicalUserPrincipal(
+    origin.cloudFrontOriginAccessIdentityS3CanonicalUserId
+  ))
+  .addAction("s3:GetObject")
+  .addResource(bucket.bucketArn)
+  .addResource(bucket.bucketArn + "/*");
+bucket.addToResourcePolicy(bucketPolicy);
+```
+
+#### CloudFront Distribution
+
+Next, Write the definition for a new CloudFront web distribution.
+
+```typescript
+const cdn = new cloudfront.CloudFrontWebDistribution(this, "CloudFront", {
+  viewerProtocolPolicy: cloudfront.ViewerProtocolPolicy.AllowAll,
+  priceClass: cloudfront.PriceClass.PriceClassAll,
+  originConfigs: [
+    {
+      behaviors: [
+        {
+          isDefaultBehavior: true,
+          maxTtlSeconds: undefined,
+          allowedMethods:
+            cloudfront.CloudFrontAllowedMethods.GET_HEAD_OPTIONS
+        }
+      ],
+      originPath: `/web`,
+      s3OriginSource: {
+        s3BucketSource: bucket,
+        originAccessIdentityId: origin.cloudFrontOriginAccessIdentityId
+      }
+    }
+  ]
+});
+```
+
+#### CloudFormation Outputs
+
+Last of all, we want to define a cloudformation output for the domain name assigned to our CloudFront distribution.
+
+```typescript
+new cdk.CfnOutput(this, "CloudFrontURL", {
+  description: "The CloudFront distribution URL",
+  value: "https://" + cdn.domainName
+});
+```
+
+With that, we have completed writing the components of our module 1 stack.  
+
+### View the generated CloudFormation Scripts
+
+From within the `cdk` folder
+
+1. npm run build
+2. cdk synth
+
+The `cdk synth` command displays the CloudFormation script that is generated based on the code you have just written.
+
+## Deploy Module 1
+
+### Bootstrapping an environment
+
+The first time you deploy an AWS CDK app into an environment (account/region), you’ll need to install a “bootstrap stack”. This stack includes resources that are needed for the toolkit’s operation. For example, the stack includes an S3 bucket that is used to store templates and assets during the deployment process.
+
+You can use the cdk bootstrap command to install the bootstrap stack into an environment:
+
+`cdk bootstrap <AWSAccountNumber>/<AWSRegion>`
+
+for example
+
+`cdk bootstrap 1234567890/eu-west-1`
+
+Switch over to the CloudFormation console (for example https://console.aws.amazon.com/cloudformation/home) and you should will the stack named `CDKToolkit` being deployed.  Wait for this deployment to complete.
+
+### Deploy the Website and Infrastructure
+
+A script has been created to simplify the deployment process for you.  The script will do the following:
```

Comment:
I think that if we use the s3-deployment module, this can just be `cdk deploy`. In general, we are aiming to provide people with an end-to-end developer experience, so they don't need "crutches" and scripts and helpers for their workflows.

Reply:
I guess we could hard code the path to the ~/Workshop/web/dist folder??